eos/b-decays/b-to-3l-nu.cc

```diff
@@ -683,6 +689,12 @@ namespace eos
             - 2 * k2 * (m_B_sq + q2)) * z_w) + k2 * (-m_B_sq + m_l_sq - q2 + sqrt(k4
             + m_B_sq_q2_diff2 - 2 * k2 * (m_B_sq + q2)) * z_w)));
 
+            if ( power_of<2>(m_B - sqrt(k2)) < q2 || q2 < 4 * m_lprime_sq
+                || power_of<2>(m_B - sqrt(q2)) < k2 || k2 < m_l_sq )
```

Comment:
See my earlier comment. This does not reflect what we have in the paper.

Reply:
done